src/app/pages/delivery-reports/delivery-reports.component.scss

```diff
@@ -0,0 +1,41 @@
+.radio-group {
+    margin-top: 20px;
+    display: block;
+    padding-left: 20px;
+    .radio-button {
+        display: inline-block;
+    }
+    textarea {
+        border-radius: 0;
+    }
+}
+.details-area {
+    padding: 20px;
+}
+
+.label-col {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+    justify-content: center;
+    border-right: 1px solid rgba(0, 0, 0, 0.12);
+    padding: 0.5rem;
+}
+
+.spaced {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    border-right: 1px solid rgba(0, 0, 0, 0.12);
+    padding: 0.5rem;
+}
+
+.status {
+    font-size: 20px !important;
```

Comment:
!important is the nighmare of styling :) - should be removed

Reply:
True, missed it